apps/web/src/routes/applet/geom_lin_trans/3d_proj_on_line/+page.svelte

```diff
@@ -11,10 +11,10 @@
   } from 'threlte-components';
   import { PrimeColor } from 'utils/PrimeColors';
 
-  const u = new Vector3(3, 4, 2);
+  const u = new Vector3(3, 4, 2); // Vector U - detached from the line
 
-  $: lineL = new Vector3(3, 2, -1);
-  $: lineDir = lineL.clone().normalize().multiplyScalar(10);
+  $: lineL = new Vector3(3, 2, -1); // Line L
```

Comment:
Hahah I appreciate you adding more comments, but this one might be a bit overkill

I meant it more as in describing what a user would see when viewing the applet, not just rewriting the code in natural language :)

Reply:
Moar comment == Moar comments 😉

Reply:
lmao fair